content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,139 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- service entry: optimize service entry updates #19305.
+- fix readiness probe inconsistency #18164.
+- xDS push optimize #18354.
+- Override locality load balancer settings per host #18406.
+- eds: do not trigger continuous full pushes when a pod is in crash loop #18574.
+- Disable redirect calls to app itself via envoy #19308.
+- add an `iptables` failure detector, default off #19534.
+- Istio: add consecutive `5xx` and gateway errors for outlier detection #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- handle http proxy while generating routes #19919.
+- Allow non-intercepting proxies `CAP_NET_BIND_SERVICE` via annotation #20378.
+- Initial support for custom Istio topology label kind/enhancement #19192.
+- `istio-iptables`: Use `iptables-restore` by default #18847.
+- Pilot-agent for in-process Ingress SDS  #18999.
+- turn on protocol sniffing for inbound #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthenticaiton` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** Auto mutual TLS is in Beta. The feature is enabled by default.
+- **Added** SDS is in Stable. SDS is enabled by default for identity provisioning.
+- **Improved** Node agent is merged with Pilot agent. This removes the requirement of pod security policy and improves security posture.
+- **Improved** Support two providers for Pilot (`istiod`) certificate: Kubernetes CA and Citadel.
+- **Improved** Support two types of JWT for CSR authentication: `first-party-jwt` and `third-party-jwt`.
+- **Improved** Provision key and cert to Prometheus through Istio Agent.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
```

Comment:
Do we care about ordering (having all `Added`s before `Improved`s, etc.)?

Reply:
Not really (c.f. https://istio.io/news/releases/1.4.x/announcing-1.4/change-notes/ for reference)